static/css/tabzilla/tabzilla.css

```diff
@@ -0,0 +1,34 @@
+#tabzilla {
```

Comment:
Can you add code comments for those changes, for when we have to merge new versions?